docker-compose.test.yml

```diff
@@ -11,7 +11,6 @@ services:
   web:
     build:
       context: .
-      target: development
```

Comment:
See the [commit message](https://github.com/department-of-veterans-affairs/vets-api/pull/17877/commits/b4cc2248ef5979faf0a13d0b6ecac3b77c276b95) about this one.